feat: add prerelease support (#214)
nearform/optic-release-automation-action: README.md

            npm run build
```

## Prerelease support

This action can be configured with the prerelease support. 

In order to do that you can configure your action adding the [npm version](https://docs.npmjs.com/cli/v9/commands/npm-version) parameters `prerelease|prepatch|preminor|premajor`:

```yml
name: release

on:
  workflow_dispatch:
    inputs:
      semver:
        description: "The semver to use"
        required: true
        default: "patch"
        type: choice
        options:
          - patch
          - minor
          - major
          - prerelease
          - prepatch
          - preminor
          - premajor
  ...
```

> It's possible to specify a custom `prerelease-prefix` if you want to have one.

Reviewer: Let's change this:

- the list of options for semver should be updated in the main part of the docs, because we have only a subset now
- in this section (as I had already mentioned before) we want to document the use of prerelease-prefix together with npm tag

Reviewer: clarify that prerelease-prefix is the input to --preid in npm publich

            npm run build
```

## Prerelease support

> This action can be configured with the prerelease support.

Reviewer: ```suggestion
This action can be used to create prereleases.
```
